<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HD Error Code Inspector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>691</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(201,52,19);</string>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_login_input">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: white;
font-size: 20px;</string>
       </property>
       <property name="maxLength">
        <number>70</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(201,52,19);</string>
       </property>
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_login_input">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: white;
font-size: 20px;</string>
       </property>
       <property name="maxLength">
        <number>70</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="login_button">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="register_page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(201,52,19);</string>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(201,52,19);</string>
       </property>
       <property name="text">
        <string>USERNAME</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(201,52,19);</string>
       </property>
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signup_button">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>220</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_register_input">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: white;
font-size: 20px;</string>
       </property>
       <property name="maxLength">
        <number>70</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_register_input">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: white;
font-size: 20px;</string>
       </property>
       <property name="maxLength">
        <number>70</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_register_input">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: white;
font-size: 20px;</string>
       </property>
       <property name="maxLength">
        <number>70</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>151</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50,50,50);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="signup_page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 14px;
    border-style: none;	
	color: white;
}

QPushButton::hover {
    border-style: solid;	
	background-color: rgb(110, 110, 110);
}
</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_page_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 14px;
    border-style: none;	
	color: white;
}

QPushButton::hover {
    border-style: solid;	
	background-color: rgb(110, 110, 110);
}</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>email_register_input</tabstop>
  <tabstop>username_register_input</tabstop>
  <tabstop>password_register_input</tabstop>
  <tabstop>signup_button</tabstop>
  <tabstop>signup_page</tabstop>
  <tabstop>email_login_input</tabstop>
  <tabstop>password_login_input</tabstop>
  <tabstop>login_page_btn</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
